scalafix-core/src/main/scala/scalafix/rewrite/NoExtendsApp.scala

```diff
@@ -0,0 +1,114 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import scalafix.syntax._
+import scalafix.util.{Whitespace => _, _}
+import scala.meta.contrib._
+import scala.meta.tokens.Token._
+
+//TODO(gabro): move the helpers somewhere else
+import NoExtendsAppSyntax._
+
+case class NoExtendsApp(mirror: Mirror) extends SemanticRewrite(mirror) {
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    def wrapBodyInMain(template: Template) =
+      ctx
+        .templateBodyTokens(template)
+        .map { body =>
+          val open =
+            ctx.addLeft(body.head, s"\n  def main(args: Array[String]) = {")
```

Comment:
I see, let's still add `: Unit =`. Although that honestly just a personal preference of mine. Unit returning methods should scream "I'm doing stuff here!"